### PR TITLE
#7 Added Footer & #2 also Fixed the visual radio button bug

### PR DESCRIPTION
**Pull request by Athyk**

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -71,6 +71,15 @@ export default class App extends Component {
         <RepositoryList
           items={state.items}
         />
+        <div className="footer">
+           <ul>
+             <li id="footer-title"><a href="https://github.com/ivanseed/gitstats"><span>Git<b>Stats</b></span></a></li>
+             <li>
+               <span>Special thanks to all contributors that made this project possible!</span>
+               <span><b><a href="https://github.com/ivanseed">ivanseed</a></b>, <b><a href="https://github.com/haroon-sheikh">haroon-sheikh</a></b>, <b><a href="https://github.com/athyk">athyk</a></b></span>
```
Comment (ivanseed) [inline src/components/app.js]: Is it possible to generate this from a GitHub API request? 
https://developer.github.com/v3/repos/#list-contributors

You would probably want to put all of this into its own separate, stateless, component.

**Comment by ivanseed** [comment]
Will check this out when I get home, but other than the one comment it is pretty much exactly what I want! Thanks for contributing!

**Comment by matankb** [comment]
Can I go ahead and work on generating the list from the github API? Or is someone else working on it?

**Comment by ivanseed** [comment]
@matankb I will create a separate ticket for this if you give me 10 minutes.

**Comment by ivanseed** [comment]
@matankb #25 all yours if you want it!

**Pull request merged**